docs/docs/build/connectors/data-source/https.md

```diff
@@ -5,40 +5,65 @@ sidebar_label: HTTPS
 sidebar_position: 25
 ---
 
-Import data from remote sources accessible via HTTP or HTTPS URLs into your Rill project.
+<!-- WARNING: There are links to this page in source code. If you move it, find and replace the links and consider adding a redirect in docusaurus.config.js. -->
 
 ## Overview
 
-The HTTPS connector allows you to import data from publicly accessible files hosted on web servers, CDNs, or cloud storage services. This is perfect for working with datasets that are regularly updated or shared publicly.
+The HTTPS connector allows you to import data from remote sources accessible via HTTP or HTTPS URLs into your Rill project. It supports publicly accessible files hosted on web servers, CDNs, or cloud storage services, making it perfect for working with datasets that are regularly updated or shared publicly.
 
+The connector supports various remote data sources:
 
-## Connect to HTTPS
+- **Public HTTP/HTTPS URLs**: Direct links to CSV, JSON, Parquet, and other data files hosted on web servers or CDNs
+- **Cloud Storage URLs**: Public links to files in Google Cloud Storage, Amazon S3, Azure Blob Storage, or other cloud providers
+- **REST API Endpoints**: Connect to REST APIs that return JSON, CSV, or other structured data formats. The connector can handle paginated responses and authenticated endpoints
 
-To connect to data via HTTPS, you have two options depending on your data source:
+The connector supports downloading and processing various file formats:
 
-1. **Use authenticated endpoints** (for private APIs or protected resources)
-2. **Use public URLs** (for publicly accessible files)
+- **CSV**: Comma-separated values files
+- **JSON**: JavaScript Object Notation files (including JSONL/NDJSON)
+- **Parquet**: Columnar data format
+- **Other formats**: Any format supported by DuckDB's file readers
 
-Choose the method that matches your data source requirements.
+
+## Create a HTTPS Connector
+
+### Public URLs
+
+For publicly accessible files, you don't need to create a connector at all. Simply specify the URL directly in your source or model configuration. If you need to use a connector (for example, to set up path prefixes), you can create one without headers:
+
+```yaml
+type: connector
+driver: https
+# No headers needed for public endpoints
+```
```

Comment:
This section is pretty confusing. 
1. The lead sentence "you don't need to create a connector at all" is then followed by a discussion of creating a connector.
2. There's no point in creating a connector file like the one shown in the example, right? 
3. The mention of "path prefixes" needs more context. If that's one of the reasons for creating a `https` connector file for public URLs, can we show an example?